pkgs/applications/audio/quodlibet/default.nix

```
@@ -35,11 +35,11 @@ python3.pkgs.buildPythonApplication rec {
 
```

I would suggest to refactor it like this, both to make it cleaner and to be able to comment the ignores:

```suggestion
  pytestFlags = stdenv.lib.optionals (xineBackend || !withGstPlugins) [
    "--ignore=tests/plugin/test_replaygain.py"
  ] ++ [
    # upstream does actually not enforce source code linting
    "--ignore=tests/quality"
    # build failure on Arch Linux
    # https://github.com/NixOS/nixpkgs/pull/77796#issuecomment-575841355
    "--ignore=tests/test_operon.py"
  ];

```

---

Done; I've pushed another commit.  Is the practice to keep it down to a single commit, though?